lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -60,8 +60,13 @@ def on_block(node)
 
         def add_wording_offense(node, message)
           docstring = docstring(node)
-          add_offense(docstring, message: message) do |corrector|
-            corrector.replace(docstring, replacement_text(node))
+
+          if node.heredoc?
+            add_offense(docstring, message: message)
+          else
+            add_offense(docstring, message: message) do |corrector|
+              corrector.replace(docstring, replacement_text(node))
```

Comment:
I wonder if this could work as
```suggestion
              corrector.replace(docstring, replacement_text(node)) unless node.heredoc?
```
Also, I am not sure if I even like my own suggestion 😄

Reply:
I just tried @Darhazer’s suggestion and it seems to work – the specs are passing. @G-Rath could you try it again please?

Reply:
Yup that's pattern is used in some of the rubocop's core cops as well

Reply:
I'm wondering if this would affect what is shown in the footer. I would guess if `add_offense` is called with a block, that means it would add to "correctable". While for heredoc it's not the case, and it should only appear as an offence, not as a autocorrectable one.
Please correct me if this assumption is wrong.

Reply:
ahh *`next`* - I was doing `return` 🤦 

Awesome, was hoping we could do something like this instead of having the if/else.